react/features/settings/components/native/ModeratorSection.tsx

```diff
@@ -35,6 +41,10 @@ const ModeratorSection = () => {
         dispatch(setFollowMe(Boolean(enabled)));
     }, [ dispatch, setFollowMe ]);
 
+    const onFollowMeRecorderToggled = useCallback((enabled?: boolean) => {
+        dispatch(setFollowMeRecorder(Boolean(enabled)));
+    }, [ dispatch, setFollowMe ]);
```

Comment:
setFolowMe -> setFollowMeRecorder 

Although I'm not sure we need it at all since it is just the action and we don't expect it to change.

Reply:
Fixed.